g2utils/um2grb2.py

```diff
@@ -2377,13 +2379,18 @@ def doFcstConvert(fname):
     :param fname: Name of the FF filename in question as a "string"
     :return: Nothing! TANGIBLE!
     """
-    global __max_long_fcst_hours__
+    global __start_long_fcst_hour__, __end_long_fcst_hour__
     
+    # calculate start hour of long fcst in multiples of 24. Why?
+    # 00 hr contains from 06 to 24 hours data.
+    # 24 hr contains from 24 to 48 hours data, and so on.
+    start_fcst_hour = (__start_long_fcst_hour__ / 24) * 24
```

Comment:
This is buggy line.
It should be " start_fcst_hour = ((__ start_long_fcst_hour__ / 24) - 1) * 24 "